src/api/api.tsx

```diff
@@ -117,6 +118,16 @@ export default {
 
   //Membership
   getMemberships: (slug: string) => IFetch<Membership[]>({ method: 'GET', url: `group/${slug}/membership/` }),
+  createMembership: (slug: string, user_id: string) =>
+    IFetch<Membership>({ method: 'POST', url: `group/${slug}/membership/`, data: { user: { user_id: user_id } } }),
+  deleteMembership: (slug: string, user_id: string) => IFetch<RequestResponse>({ method: 'DELETE', url: `group/${slug}/membership/${user_id}/` }),
+  updateMembership: (slug: string, user_id: string, data: { membership_type: MembershipType }) =>
+    IFetch<Membership>({ method: 'PUT', url: `group/${slug}/membership/${user_id}/`, data }),
+
+  //Group
+  getGroups: () => IFetch<Group[]>({ method: 'GET', url: `group/` }),
+  getGroup: (slug: string) => IFetch<Group>({ method: 'GET', url: `group/${slug}/` }),
+  updateGroup: (data: Group) => IFetch<Group>({ method: 'PUT', url: `group/${data.slug}/`, data }),
```

Comment:
```suggestion
  updateGroup: (slug: string, data: Partial<Group>) => IFetch<Group>({ method: 'PUT', url: `group/${slug}/`, data }),
```

Tror det er bedre å dele opp slug og data, samt gjøre data til Partial slik at man kan sende inn oppdatering av kun beskrivelse hvis man ønsker det (sjekk med backend at de tillater delvis PUT).

Reply:
Virker ikke som det er tillatt dessverre

Reply:
Det er vel derfor vi ville brukt Patch...

Reply:
Virker ikke som det er tillatt